dvc/path/azure.py

```diff
@@ -0,0 +1,9 @@
+from dvc.path import BasePathInfo, Schemes
+
+
+class AzurePathInfo(BasePathInfo):
```

Comment:
Can we inherit from S3PathInfo too? Or maybe even we should create a common base class and inherit from it in azure, s3 and gs path infos.